### PR TITLE
feat: NACEBEL

### LINKED ISSUE #47: feat: add NACEBEL 2003 & 2008
https://economie.fgov.be/sites/default/files/Files/Entreprises/KBO/conversiontable-Nacebel-2003-2008.xls

Add both versions and conversion from 2003 -> 2008

### PR DESCRIPTION
**Pull request by nyoungstudios**
## Description
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
- adds NACEBEL 2003 and 2008, plus mapping for NACEBEL 2008 to NACE2
- closes #47 

## Types of changes

What types of changes does your code introduce:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: pyisic/_standards/nacebel2003/nacebel2003.py [hunk 1]
```diff
@@ -0,0 +1,5464 @@
+from ...types import Classification, Standard, Standards
```
Comment (JSPromisel) [inline pyisic/_standards/nacebel2003/nacebel2003.py]: Add docstring like:
```
"""`NACEBEL2003 Standard <link to xlsx file>`_.
"""
```

File: pyisic/_standards/nacebel2008/nacebel2008.py [hunk 2]
```diff
@@ -0,0 +1,3976 @@
+from ...types import Classification, Standard, Standards
+
+NACEBEL2008 = Standard(
```
Comment (JSPromisel) [inline pyisic/_standards/nacebel2008/nacebel2008.py]: Take a look at the [NACE2 standard](https://github.com/sayari-analytics/pyisic/blob/main/pyisic/_standards/nace2/nace2.py) I think some of these qualify as different categories e.g. Section vs Class)

File: pyisic/_standards/nacebel2008/nacebel2008_to_nacebel2003.py [hunk 3]
```diff
@@ -0,0 +1,3649 @@
+from ...types import Concordance, Standards
```
Comment (JSPromisel) [inline pyisic/_standards/nacebel2008/nacebel2008_to_nacebel2003.py]: Remove this concordance

**Review by JSPromisel — CHANGES_REQUESTED** [review]
This looks good thanks. Here are the changes:

- remove the unnecessary reversed concordances
- add docstrings of the same form as the other standards/concordances
- Add new concordances to the [test_all_concordances](https://github.com/sayari-analytics/pyisic/blob/main/tests/test_all_concordances.py) test suite
- Look at adding categories to the standards (e.g. class, section)

File: pyisic/_standards/nacebel2008/nacebel2008_to_nace2.py [hunk 4]
```diff
@@ -0,0 +1,490 @@
+from ...types import Concordance, Standards
```
[the remark this answers is not in the document]
Comment (nyoungstudios) [reply pyisic/_standards/nacebel2008/nacebel2008_to_nace2.py]: I'm not sure where to find a link like this, but I just added the same excel document link as the other ones. And I did add a statement mentioning how this was derived. Let me know if that is good or not

**Review by JSPromisel — CHANGES_REQUESTED** [review]
Thanks, these changes look good. Can you just add the following:

- add new concordances to `test_minimum_concordance`
- add docstrings to both new standards as well

**Comment by nyoungstudios** [comment]
@JSPromisel yep, I've updated those. I had accidentally deleted the docstring for the new standards when I regenerated it with the categories

**Review by JSPromisel — APPROVED** [review]

**Pull request merged**